docs/installation.rst

```diff
@@ -124,21 +124,7 @@ versions for dependencies. ::
     (mssenv) $ msui
 
 
-Update
-------
-
-Builtin Update
-..............
-
-Since version 5.0 we provide a feature for updating MSS by the UI or the command line
-After you started the MSS UI it informs you after a while if there is a new update available.
-From the command line you can trigger this update feature by ::
-
-    (mssenv) $ msui --update
-
-
-
-Other Methods
+Update Methods
 .............
```

Comment:
According to the rst docs I could find, a section underline must be at least as wide as the title.